Corvus.Extensions.Samples/TaskExtensionsSample.dib

```diff
@@ -0,0 +1,72 @@
+#!markdown
+
+## TaskExtensions - Sample notebook
+
```

Comment:
Our other sample notebooks have a section titled "How to use .NET Interactive notebooks" at the beginning - we should include that here for consistency.

Reply:
Never mind, I've just found the pull request that removes all of these sections and puts them in a central README.md 🙂